File: Programming_Fundamentals_With_Python/General-Exercises/Gladiator Expenses.py
lost_fights = int(input()) # 0 / 1000
helmet_price = float(input()) # 0 / 1000
sword_price = float(input()) # 0 / 1000
shield_price = float(input()) # 0 / 1000
armor_price = float(input()) # 0 / 1000
expenses = 0
shield_brakes = 0
for brake in range(1, lost_fights + 1):
    if brake % 2 == 0:
        expenses += helmet_price
    if brake % 3 == 0:
        expenses += sword_price
        if brake % 2 == 0:
            shield_brakes += 1
            expenses += shield_price
            if shield_brakes % 2 == 0:
                expenses += armor_price

print(f"Gladiator expenses: {expenses:.2f} aureus")

File: Javascript/JSSyntaxFundamentals/moreExercises/gladiatorExpenses.js
function solve(lostFights, helmetPrice, swordPrice, shieldPrice, armorPrice){
    let expenses = 0;
    let shieldBreaks = 0;
    for (let i = 1; i <= lostFights; i++){
        if (!(i % 2)){
            expenses += helmetPrice;
            if (!(i % 3)) {
                expenses += shieldPrice;
                shieldBreaks += 1;
                if (!(shieldBreaks % 2)){
                    expenses += armorPrice;
                }
            }
        }
        if (!(i % 3)){
            expenses += swordPrice;
        }

    }
    return `Gladiator expenses: ${expenses.toFixed(2)} aureus`
}

console.log(solve(23,
    12.50,
    21.50,
    40,
    200

))